src/main/java/nextstep/subway/line/domain/Section.java

```diff
@@ -105,4 +107,20 @@ public boolean equalUpStation(Station station) {
     public boolean equalDownStation(Station station) {
         return downStation.equals(station);
     }
+
+    public void addVertexAndEdgeToGraph(WeightedMultigraph<Station, DefaultWeightedEdge> graph) {
+        addVertexToGraph(graph, upStation);
+        addVertexToGraph(graph, downStation);
+        addEdgeToGraph(graph);
+    }
+
+    private void addVertexToGraph(WeightedMultigraph<Station, DefaultWeightedEdge> graph, Station station) {
+        if(graph.containsVertex(station)) return;
```

Comment:
잘 알고 계시긴 할건데 컨벤션 한번 짚고 가시죠 😆 
```suggestion
        if (graph.containsVertex(station)) return;
```